vagrant/Vagrantfile

```diff
@@ -1,6 +1,7 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+disable_gui = ENV['VESPA_DISABLE_VAGRANT_GUI']
```

Comment:
Consider rename to VESPA_VAGRANT_DISABLE_GUI to use the same prefix as the other variables